src/main/kotlin/org/rust/ide/annotator/RsHighlightingAnnotator.kt

```diff
@@ -81,6 +90,9 @@ class RsHighlightingAnnotator : AnnotatorBase() {
 
     private fun highlightIdentifier(element: PsiElement, parent: RsElement, holder: AnnotationHolder): RsColor? {
         return when {
+            parent is RsMetaVarIdentifier || parent is RsMacroBinding || parent is RsMacroPattern -> {
+                RsColor.MACRO
```

Comment:
And now macro meta variables are highlighted in the same color as other parts of macros

Reply:
Yeah. Metavariables are not variables--so that's the intent.

If we want different parts of macros have different colors, we need new RsColors--which we will add in the next PR.